.travis.yml

```diff
@@ -0,0 +1,83 @@
+language: python
+python:
+- '3.6'
+dist: trusty
+# We're going to ignore all branches except:
+branches:
+  only:
+  - master
+  - production
+
+# We will need these services:
+services:
+- docker
+- postgresql
+
+sudo: required
+
+# We need PostgreSQL 10 installed
+addons:
+  postgresql: "10"
```

Comment:
is this ok, given the change to 9.6 in your other branch?

Reply:
I have fixed this in my branch, we will continue to use 10. A specific step was added in the dockerfile to install Pg10.

Reply:
I working to get this sorted before Thursday